src/components/TimerContainer.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react';
+import { publish, subscribe } from './Event';
+
+function TimerContainer(props) {
+  const { questionNum, quiz } = props;
+  const [count, setCount] = useState(Number(quiz.timerVal));
+
+  const updateCount = () => {
+    if (count > 1) {
+      setCount((prevState) => prevState - 1);
```

Comment:
Why have we needed to use the prevState here? Is it required?

Reply:
Change 'prevState' to 'lastCount'. We used previous last count value to set new count.